test/testpushnotifications.cpp

```diff
@@ -2,10 +2,30 @@
 #include <QVector>
 #include <QWebSocketServer>
 #include <QSignalSpy>
+#include <qsignalspy.h>
 
+#include "accountfwd.h"
 #include "pushnotifications.h"
 #include "pushnotificationstestutils.h"
 
+bool verifyCalledOnceWithAccount(QSignalSpy &spy, OCC::AccountPtr account, bool wait = false)
```

Comment:
`const OCC::AccountPtr &account`

Also I'd remove the `wait` parameter and let the responsibility of this to the caller, it's so much more readable in the test, I mean compare this:
```cpp
QVERIFY(verifyCalledOnceWithAccount(spy, account, true));
```
With this:
```cpp
QVERIFY(spy.wait());
QVERIFY(verifyCalledOnceWithAccount(spy, account));
```
Intent is much clearer in the second version

Reply:
Yes true. I change it.